website/src/pages/use-cases/code-reuse.tsx

```diff
@@ -0,0 +1,310 @@
+            description:
+                'Find existing code libraries for reuse and avoid spending time on problems a teammate already solved for a more secure and coherent codebase.',
+        }}
+        className="use-cases-page"
+        hero={
+            <section className={`${styles.useCaseHeader} navbar-light`}>
+                <div className="container">
+                    <div className="row">
+                        <div className="col-md-7 my-7">
+                            <BackButtonBold href="/use-cases" text="USE CASES" />
+                            <h1 className="display-2 font-weight-bold mb-4">Find and use code that already exists</h1>
+                            <div className="display-4 font-weight-normal mb-5">
+                                Identify existing code libraries for reuse and use innersourcing to avoid spending time
+                                on problems a teammate already solved.
+                            </div>
+                            <div className="d-flex flex-column flex-lg-row pt-1">
+                                <Link
+                                    className="btn btn-primary mr-lg-3 mb-lg-0 mb-3 w-md-100"
+                                    to="/demo"
+                                    title="Request a Demo."
+                                >
+                                    Request a demo
+                                </Link>
+                                <Link
+                                    className="btn btn-outline-primary w-md-100"
+                                    to="/get-started"
+                                    title="Try Sourcegraph."
+                                >
+                                    Try Sourcegraph now
+                                </Link>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        }
+    >
+        <div className="bg-white">
+            <ContentSection className="my-7">
+                <ThreeUpText title="Identify, resolve, and monitor with confidence" items={threeUpTextItems} />
+            </ContentSection>
+        </div>
+
+        <div className="bg-gradient-venus-radial">
+            <ContentSection className="my-7">
+                <div className="row flex-column flex-lg-row justify-content-between px-0">
+                    <div className="p-lg-0 col-xl-6 col-lg-7 px-4">
+                        <h1 className="mb-4 font-weight-bold max-w-600">
+                            Finding reliable, reusable code is tedious and inefficient
+                        </h1>
+                        <p>
+                            Current tools don’t fully enable teams to innersource and use existing code to develop a
+                            more secure and coherent codebase.
+                            <span className="font-weight-bold"> What does that mean for you and your team?</span>
```

Comment:
@st0nebraker: We just received [this comment](https://docs.google.com/document/d/18dNP6QihejP8CVSm-tdmbH20D5beFYlGXyBme0286U4/edit?disco=AAAAYfEfxiQ) from Marketing to unbold this line ("What does that mean for..."). I am working on updating instances in the other Use Case pages at https://github.com/sourcegraph/about/pull/5297. Just pinning here for consistency!

Reply:
Thanks, @zlonko ! Fixing it on this branch to be proactive 👍